br/pkg/restore/log_client/migration.go

```diff
@@ -245,11 +245,11 @@ func (wm *WithMigrations) Metas(metaNameIter MetaNameIter) MetaMigrationsIter {
 	})
 }
 
-func (wm *WithMigrations) Compactions(ctx context.Context, s storage.ExternalStorage) iter.TryNextor[*backuppb.LogFileSubcompaction] {
+func (wm *WithMigrations) Compactions(ctx context.Context, s storage.ExternalStorage, shiftStartTS, restoreTS uint64) iter.TryNextor[*backuppb.LogFileSubcompaction] {
```

Comment:
Is it possible to add `shiftStartTS` to the `WithMigrations`? I think in this context we may usually require this TS. Also it seems `WithMigrations` has a field of restored ts.

Reply:
already have, fixed.